gateway/handler_success.go

```diff
@@ -327,7 +327,7 @@ func (s *SuccessHandler) ServeHTTPWithCache(w http.ResponseWriter, r *http.Reque
 	if !s.Spec.VersionData.NotVersioned && versionDef.Location == "url" && versionDef.StripPath {
 		part := s.Spec.getVersionFromRequest(r)
 
-		log.Info("Stripping version from url: ", part)
+		log.Debug("Stripping version from url: ", part)
```

Comment:
Perhaps use `URL` (uppercase) as we do in other similar log messages?

Reply:
@matiasinsaurralde done. PTAL , Although its outside the scope of the PR no harm having it.